README.md

```diff
@@ -66,12 +93,123 @@ You'll also need to create some (human) users.  The email address of the new use
+sudo systemctl stop mysql.service redis_6379.service redis-server.service
+
+# Create docker images and launch the service
+docker-compose -f deployments/local/docker-compose.yml up -d
+
+# Initialise the database
+mysql -u $DATABASE_USER -p -h 127.0.0.1 $DATABASE_NAME < packages/backend-core/seed/initial-database.sql
+./bin/osmod migrate
+```
+
+To shut down the service and delete all your containers:
+
+```bash
+docker-compose -f deployments/local/docker-compose.yml down
+```
+
+And to see what the container is doing:
 
 ```bash
-./bin/osmod ...
+docker-compose -f deployments/local/docker-compose.yml logs
 ```
 
+#### Adding users
+
+To actually do anyting, you'll need to create some users.  You'll need at least
+
+* a human user so you can log in and view comments
+
+* a service user that knows how to talk to a PerpectiveAPI proxy server.
+
+Add human users with the following command:
+
+```bash
+./bin/osmod users:create --group general --name "Name" --email "email@gmail.com"
+```
+
+Human users get authenticated using Google's OAuth server, so the email address must
+correspond to a google account.
+
+Add the proxy service user with the following command:
+
+```bash
+./bin/osmod users:create --group service --name "PerspectiveProxy" --endpoint=<proxy URL>
+```
+
+where `<proxy URL>` is the URL of [The Perspective API proxy] you plan on using.
+
+### The Perspective API proxy
+
+OSMod is designed to talk to the Perspective API via a proxy server: the
+[Perspective API proxy](https://github.com/conversationai/perspectiveapi-proxy).
+
+There is a preconfigured proxy server set up for general use.
+The URL of this proxy is [https://osmod-assistant.appspot.com/].  To access it,
+you'll need to get a suitable authentication token from
+[the Perspective API team](https://www.perspectiveapi.com/#/).  In this case you
+need to set your service user's proxy URL to
+`https://osmod-assistant.appspot.com/api/score-comment`.
+
+Alternatively, if you have been given direct access to the Perspecitve API, you
+can create a Perspective API key in a Google Cloud project and then run
+a local instance of the proxy as described in the
+[Perspective API proxy documentation](https://github.com/conversationai/perspectiveapi-proxy/blob/master/README.md).
+E.g.:
+
+```bash
+export GOOGLE_CLOUD_API_KEY=<API Key>
+export AUTH_WHITELIST=$GOOGLE_SCORE_AUTH
+export ATTRIBUTE_REQUESTS
+read -r -d '' ATTRIBUTE_REQUESTS << EOM
+{
+  "ATTACK_ON_AUTHOR": {},
+  "ATTACK_ON_COMMENTER": {},
+  "INCOHERENT": {},
+  "INFLAMMATORY": {},
+  "OBSCENE": {},
+  "OFF_TOPIC": {},
+  "SPAM": {} ,
+  "UNSUBSTANTIAL": {},
+  "LIKELY_TO_REJECT": {},
+  "TOXICITY": {},
+  "TOXICITY_FAST": {}
+}
+EOM
+
+cd $PERSPECTIVEAPI_PROXY_LOCATION
+yarn install
+PORT=8081 ATTRIBUTE_REQUESTS='{ "TOXICITY": {} }' yarn run watch
+```
+
+TODO: Need fixes to proxy and osmod:
+ATTACK_ON_PUBLISHER is in proxy's default list and osmod UI, but doesn't work.
+SUMMARY_SCORE in osmod UI, but doesn't work.
```

Comment:
summary score should be a field inside each attribute's response, not an attribute in its own right.